governance/polls/Ratification Poll - Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Ratification Poll for Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022
+summary: Regarding the offboard of the Content Production CU, this poll is for removing their wallet from the recognized multi-sig list.
+discussion_link: https://forum.makerdao.com/t/mip47c3-sp1-dissolution-of-content-production-mkt-001-multi-sig-request/13991
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-11T16:00:00
+end_date: 2022-04-25T16:00:00
+---
+# Ratification Poll for Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, April 11 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP47c3-SP1 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*Now that the Content Production Core Unit has been offboarded, their wallet should be removed from the recognized multi-sig list. Currently, no funds are available in the multi-sig, but it can still claim ~3.8k DAI from their canceled budget stream.
+
+Seeing as the Content Production Core Unit is no longer an active Core Unit for the DAO, there is only downside to leaving their multi-sig Governance recognized. Given the small amount of funds that can still be claimed, there is little danger to publicly announce our intentions to close the loop with the DssVest stream. However, this process is clearly vulnerable to front running until the procedure for reclaiming funds and permissions is improved.
+
+If this subproposal is accepted, there will be the technical work of claiming the budget funds and returning them to the surplus buffer, to be completed in a future executive proposal.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/25c4b48d8e0f6d2ccee517df0555f0766feff8a5/MIP47/MIP47c3-Subproposals/MIP47c3-SP1.md)
+* [Latest Proposal Version](Can't Find One)
```

Comment:
That's what I corrected :)

Here it is: https://mips.makerdao.com/mips/details/MIP47c3SP1